src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealServiceBase.java

```diff
@@ -148,7 +149,8 @@ SscsCaseData convertAppealToSscsCaseData(SyaCaseWrapper appeal) {
 
         String postCode = resolvePostCode(appeal);
         String firstHalfOfPostcode = getFirstHalfOfPostcode(postCode);
-        RegionalProcessingCenter rpc = regionalProcessingCenterService.getByPostcode(firstHalfOfPostcode);
+        boolean isIba = appeal.getBenefitType().getCode().equals(Benefit.INFECTED_BLOOD_COMPENSATION.getShortName());
```

Comment:
would it make sense to have a helper method so it's like `appeal.getBenefitType().isIba()`

Reply:
can do